tests/integration/constraints/test_tabular.py

```diff
@@ -148,3 +149,28 @@ def test_ScalarRange():
     # Assert
     assert sampled.column.min() >= 5
     assert sampled.column.max() <= 11
+
+
+def test_ScalarRange_conditions():
+    """Test ``ScalarRange`` with conditions."""
```

Comment:
minor, but can we add to the description to say why we needed to check this (to make sure the conditions weren't being altered by the constraint transformation)